translations/ja.json

```diff
@@ -954,64 +954,64 @@
 	"scheduled stop is not supported on the none driver, skipping scheduling": "none ドライバーでは予定停止がサポートされていません (予約をスキップします)",
 	"service {{.namespace_name}}/{{.service_name}} has no node port": "サービス {{.namespace_name}}/{{.service_name}} は NodePort がありません",
 	"startup failed": "起動に失敗しました",
-	"stat failed": "stat が失敗しました",
+	"stat failed": "stat に失敗しました",
 	"status json failure": "status json に失敗しました",
 	"status text failure": "status text に失敗しました",
-	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数の数（{{.ArgCount}}）が多すぎます。\\n使用方法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
-	"tunnel creates a route to services deployed with type LoadBalancer and sets their Ingress to their ClusterIP. for a detailed example see https://minikube.sigs.k8s.io/docs/tasks/loadbalancer": "tunnel によってタイプが LoadBalancer なサービスへのルーティングが作成され、Ingress をサービスの ClusterIP へと向けさせます。より詳細な例は以下を参照してください。https://minikube.sigs.k8s.io/docs/tasks/loadbalancer",
-	"tunnel makes services of type LoadBalancer accessible on localhost": "tunnel によってタイプが LoadBalancer なサービスが localhost からアクセス可能になります",
-	"unable to bind flags": "フラグをバインドすることができませんでした",
-	"unable to daemonize: {{.err}}": "",
-	"unable to delete minikube config folder": "minikube の設定フォルダーを削除できませんでした",
-	"unable to set logtostderr": "logtostderr を設定することができませんでした",
-	"unpause Kubernetes": "Kubernetes を再開させます",
-	"unset failed": "取り消しが失敗しました",
-	"unsets PROPERTY_NAME from the minikube config file.  Can be overwritten by flags or environmental variables": "minikube の設定ファイルから PROPERTY_NAME の値を取り消します。フラグ、あるいは環境変数で上書き可能です",
-	"unsets an individual value in a minikube config file": "minikube の設定ファイルの個々の値を取り消します",
-	"unsupported or missing driver: {{.name}}": "サポートしていない、あるいは不足しているドライバーです: {{.name}}",
+	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}} 個) が多すぎます。\\n使用法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```

Comment:
Just comment.
s/toom any/too many/ right?

Reply:
@atoato88 right. It looks typo and I'll make an issues to fix it.